llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -106,6 +106,8 @@ static const RISCVSupportedExtension SupportedExtensions[] = {
     {"xtheadmempair", {1, 0}},
     {"xtheadsync", {1, 0}},
     {"xtheadvdot", {1, 0}},
+    {"xtheadvector", {1, 0}},
+    {"xtheadzvamo", {1, 0}},
```

Comment:
I don't know we need this. IIRC, C906/C910 didn't implement this extension. CC @zixuan-wu @rjiejie

Reply:
HI, Pengcheng,
In fact, C906/C910 support the `Zvamo` extension related instructions. For details [here](https://www.xrvm.cn/product/xuantie/C906?spm=a2cl5.14290816.0.0.4768AjP2AjP2z2). As I have already described in [the spec](https://github.com/T-head-Semi/thead-extension-spec/blob/master/xtheadvector.adoc), it has only been renamed from `Zvamo` to `XTheadZvamo`. BTW, Here is a [binutils patch](https://sourceware.org/pipermail/binutils/2023-November/130677.html) that has been tested and reviewed for reference.